docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/data-imports/how-to-manage-merchant-custom-prices-for-products-via-data-import.md

```diff
@@ -0,0 +1,45 @@
+You can do this by [adding merchant custom prices via the Back Office](https://docs.spryker.com/docs/scos/user/back-office-user-guides/202108.0/catalog/products/abstract-products/creating-abstract-products-and-product-bundles.html#reference-information-defining-prices), or by data import. In this document, we consider how you can manage the merchant custom prices with data import.
+
+## Data importers for merchant custom prices
+
+In the Spryker B2B Demo Shop, there are two data importers used for managing the merchant custom prices:
+
+- [merchant_relationship.csv](https://github.com/spryker-shop/b2b-demo-shop/blob/master/data/import/common/common/merchant_relationship.csv) containing information about merchants and merchant relation keys. The merchant relation keys are identifiers of the relations between sellers and buyers.
+- [price-product-merchant-relationship.csv](https://github.com/spryker-shop/b2b-demo-shop/blob/master/data/import/common/DE/price_product_merchant_relationship.csv) containing specific product prices for individual merchant relations.
+
+## Assigning merchant custom prices via data import
+
+To assign special product prices to merchants, you can do the following:
+
+1. Define the merchant relation keys in the `merchant_relationship.csv` file. See the [example in the Spryker B2B Demo Shop](https://github.com/spryker-shop/b2b-demo-shop/blob/master/data/import/common/common/merchant_relationship.csv).
+2. Run 
+   ```Bash
+   console data:import merchant-relationship
+   ```
+3. Create a separate data import .CSV files for each merchant relationship. For example, you could have a file `price-product-merchant-relationship-mr001.csv`, where `mr001` is the merchant relation key you defined in the previous step. In this file, you would then specify product prices for this specific merchant.
+4. Then, you can do one of the following:
+- Set specific default prices for this merchant manually.
+That means, if you provide say 20% discount on a product for merchant relation `mr001` and change the default price for the product in the [product price file](https://docs.spryker.com/docs/scos/dev/data-import/{{site.version}}/data-import-categories/catalog-setup/pricing/file-details-product-price.csv.html), you would have to go to the `price-product-merchant-relationship-mr001.csv` file and change the price for this merchant as well. Because changing the general default product price does not automatically change the merchant custom price for this product. Thus, with every change of the default product price, you have to manually change the every merchant custom price of this product.
+This option can work for a relatively small amount of products and merchants you provide the specific prices for. For big amount of this data, to avoid too much manual work, we recommend considering the next option.
+- Create a custom script that would automatically adjust merchant-specific prices upon the default price change.
```

Comment:
As a side note, we could build a relatively small script that will do that until we cover this problem properly in the Product.
We could have this script under "Spryker Utils" lol